extensions/scarb-doc/src/docs_generation/markdown/book_toml.rs

```diff
@@ -0,0 +1,29 @@
+use indoc::formatdoc;
+
+use crate::AdditionalMetadata;
+
+#[allow(dead_code)]
+pub(super) fn generate_book_toml_content(package_metadata: &AdditionalMetadata) -> String {
```

Comment:
```suggestion
pub fn generate_book_toml_content(package_metadata: &AdditionalMetadata) -> String {
```

why not just do not re-export this item in super module?

Reply:
then this `(super)` part has no effect so you do not need it 😃 

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=dd44beed411270287a8fccc5b5826383

```rust
mod foo {
    mod bar {
        pub fn b() -> u64 {
            42
        }
    }

    pub fn f() -> u64 {
        bar::b()
    }
}

fn main() {
    println!("{}", foo::f());
    println!("{}", foo::bar::b());
}
```

```
   Compiling playground v0.0.1 (/playground)
error[E0603]: module `bar` is private
  --> src/main.rs:15:25
   |
15 |     println!("{}", foo::bar::b());
   |                         ^^^  - function `b` is not publicly re-exported
   |                         |
   |                         private module
   |
note: the module `bar` is defined here
  --> src/main.rs:2:5
   |
2  |     mod bar {
   |     ^^^^^^^

For more information about this error, try `rustc --explain E0603`.
error: could not compile `playground` (bin "playground") due to 1 previous error
```